utt/libutt/test/TestParallelPairing.cpp

```diff
@@ -29,19 +29,21 @@ int main(int argc, char *argv[]) {
   std::vector<G1> a(numIters);
   std::vector<G2> b(numIters);
 
-#ifdef USE_MULTITHREADING
-#pragma omp parallel for
```

Comment:
This pragma is used to run the for loop on parallel threads using the omp library. I think that the compiler flag needed to link against omp differs between the two compilers.

Reply:
There's a TODO comment for fixing it